frontend/doc-site/02_Find-Published-Data.mdx

```diff
@@ -6,4 +6,6 @@ Below, you will learn how to find data of interest on the CellxGene Data Portal.
 
 ## INSPECT, SEARCH, AND FILTER DATA
 
-//EMBED THE INSPECT DATA, SEARCH AND FILTER TUTORIAL HERE - there is page content, but I think it needs to be updated based on CC work since Q4 2021? We should just make a slideshow w the updated information.
+Check out the tutorial below to discover how inspect, search, and filter data.
+
+<EmbeddedGoogleSlides src="https://docs.google.com/presentation/d/1FFeHuca_pqOjmbV9H4MaoYhdspIahCNcUrVQrCdqK7I/edit?usp=sharing" />
```

Comment:
@MaximilianLombardo it looks like there are a bunch of additional slides in the tutorial. Are you planning to remove those?

Reply:
yes - thanks for flagging - just removed these and also corrected the title of the slides